pkg/universe.dagger.io/python/image.cue

```diff
@@ -0,0 +1,22 @@
+package python
+
+import (
+	"universe.dagger.io/docker"
+)
+
+#Image: {
+	pythonVersion: =~#"\d+(?:\.\d+)"# & (*"3.10" | string)
+	alpine:        bool
+	_source:       string
+	*{
+		alpine:  true
+		_source: "python:\(pythonVersion)-alpine"
+	} | {
+		alpine:  false
+		_source: "python:\(pythonVersion)"
+	}
+
+	docker.#Pull & {
+		source: "python:\(pythonVersion)-alpine"
```

Comment:
This needs to use `_source`.

Reply:
Ah shoot yes, this was a troubleshooting step I didn't notice stayed in. Will update.